op-batcher/batcher/service.go

```diff
@@ -48,7 +49,13 @@ type BatcherConfig struct {
 	// For throttling DA. See CLIConfig in config.go for details on these parameters.
 	ThrottleThreshold, ThrottleTxSize          uint64
 	ThrottleBlockSize, ThrottleAlwaysBlockSize uint64
+	ThrottleThresholdMultiplier                float64
 	ThrottlingEndpoints                        []string
+
+	ThrottleControllerType config.ThrottleControllerType
+	ThrottlePidConfig      *config.PIDConfig
+
+	PreferLocalSafeL2 bool
```

Comment:
This was removed recently, you may want to update your branch with latest `develop`.